app/src/main/java/io/homeassistant/companion/android/util/vehicle/NativeModeActionStrip.kt

```diff
@@ -1,39 +1,36 @@
-package io.homeassistant.companion.android.util.vehicle

```

Comment:
The diff in this file has become hard to review, can you please update this so only the changes are visible? I think this might have to do with the line endings on the file being changed?

Reply:
Sorry, it's `./gradlew ktlintFormat`. Fixed it